plugins/csv_dataset.cc

```diff
@@ -897,7 +908,7 @@ struct CsvDataset::Itl: public TabularDataStore {
                 //threadAccum.emplace_back(std::move(lineEntry));
             };
 
-        forEachLineBlock(stream, onLine, lineOffset, config.limit);
```

Comment:
forEachLineBlock does not return error messages.  The caller is responsible to know how many lines were skipped (because of header, offset) and return the proper line number in error messages (see actualLineNum).  I changed it because it was too misleading since the code do not actually skip any lines.